### ISSUE TITLE
elpy not finding jedi in virtualenv

### ISSUE THREAD

**Issue by hmelman** [jorgenschaefer/elpy#1617]
# Summary

I'm finding that goto definitions is no longer working and I'm assuming it's because I recently upgraded elpy. According to `elpy-config` it can no longer find `jedi` though it is installled.

I'm on macOS 10.14.5, running the latest version of the macport of emacs, GNU Emacs 26.2 (build 1, x86_64-apple-darwin18.5.0, Carbon Version 158 AppKit 1671.4) of 2019-06-08

my virtualenv is faenv and its got jedi 0.14.0 installed

```
hmelman@fadmin$ ./faenv/bin/pip --version
pip 19.1.1 from /Users/hmelman/src/fadmin/faenv/lib/python3.7/site-packages/pip (python 3.7)
hmelman@fadmin$ ./faenv/bin/pip install --upgrade jedi 
Requirement already up-to-date: jedi in ./faenv/lib/python3.7/site-packages (0.14.0)
Requirement already satisfied, skipping upgrade: parso>=0.3.0 in ./faenv/lib/python3.7/site-packages (from jedi) (0.4.0)
```

I saw some recent commits about changing how elpy wants jedi configured and uses the rpc. I'm guessing that's my issue. Since I'm on a mac my global python config is 2.7 and I really don't want to use that but I updated jedi there too and still get the same elpy-config shown. Both globally and in faenv python3 points to 3.7.3 and I'd rather use that.

```
hmelman@fadmin$ gpip install --upgrade jedi
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Collecting jedi
  Using cached https://files.pythonhosted.org/packages/68/42/6309f3871b2f8361764ac5b2fe6719f9c6e6561d9307d8cecda319cf5843/jedi-0.14.0-py2.py3-none-any.whl
Collecting parso>=0.3.0 (from jedi)
  Downloading https://files.pythonhosted.org/packages/68/59/482f5a00fe3da7f0aaeedf61c2a25c445b68c9124437195f6e8b2beddbc0/parso-0.5.0-py2.py3-none-any.whl (94kB)
    100% |████████████████████████████████| 102kB 1.8MB/s 
Installing collected packages: parso, jedi
  Found existing installation: jedi 0.9.0
    Uninstalling jedi-0.9.0:
      Successfully uninstalled jedi-0.9.0
Successfully installed jedi-0.14.0 parso-0.5.0
```

# Steps to reproduce

# My configuration
## OS
I'm on macOS 10.14.5, running the latest version of the macport of emacs, GNU Emacs 26.2 (build 1, x86_64-apple-darwin18.5.0, Carbon Version 158 AppKit 1671.4) of 2019-06-08

## Result of `(elpy-config)`
```
Elpy Configuration

Virtualenv........: faenv (/Users/hmelman/src/fadmin/faenv)
RPC Python........: 3.7.3 (/Users/hmelman/src/fadmin/faenv/bin/python)
Interactive Python: python (/Users/hmelman/src/fadmin/faenv/bin/python)
Emacs.............: 26.2
Elpy..............: 1.30.0
Jedi..............: Not found (0.14.0 available)
Rope..............: 0.14.0
Autopep8..........: 1.4.4
Yapf..............: 0.27.0
Black.............: 19.3b0
Syntax checker....: flake8 (/Users/hmelman/src/fadmin/faenv/bin/flake8)

The jedi package is not available. Completion and code navigation will
not work.

[run] python -m pip install jedi

```

## Elpy configuration in my init.el
```
(when (require 'elpy nil t)
  (elpy-enable))

(pyvenv-activate "~/src/fadmin/faenv")
```

**Comment by galaunay** [comment]
There is an recent [issue](https://github.com/davidhalter/jedi/issues/1348) with jedi 0.14.
Would it fix your issue if you install parso==0.5 in the virtualenv ?

**Comment by hmelman** [comment]
Yes, that did fix my issue. I did have to restart emacs to make it work but that did it.

To be clear:
- I had a running emacs, `elpy-config` showed as above. 
- In a terminal window outside emacs I upgrade parso. 
- In emacs I did `elpy-config` and it showed it found jedi, but `xref-find-definition` didn't work.
- I tried `elpy-enable` and it still didn't work
- I restarted emacs and it worked just fine.

Thanks very much.
